en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/resiliency/circuit-breakers.md

```diff
@@ -0,0 +1,82 @@
+# Circuit Breakers
+
+In case of backend failure it is always better to fail quickly and relieve the back pressure from the actual backend, so it can recover quickly. The Choreo Connect supports various types of distributed circuit breakers offered by the Envoy. 
+Based on the circuit breaker configuration, the router (i.e. Envoy) will open the circuit and handle the consequent request by itself till the backend recovers. 
+
+## Enable circuit breakers for API endpoints
+
+!!! note
+    Circuit breakers are enabled by default and have modest default values.
```

Comment:
```suggestion
    Circuit breakers are enabled by default and have moderate default values.
```
Can we use moderate or something similar here?

Reply:
yes. that would be fine